addons/airway/functions/fnc_treatmentAdvanced_RemoveAirwayItemLocal.sqf

```diff
@@ -0,0 +1,36 @@
+#include "script_component.hpp"
+/*
+ * Author: Miss Heda
+ * Removing Guedeltubus & KingLT
+ *
+ * Arguments:
+ * 0: Medic <OBJECT>
+ * 1: Patient <OBJECT>
+ * 2: Item <STRING>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player, cursorTarget] call kat_airway_fnc_treatmentAdvanced_RemoveGuedeltubusLocal;
```

Comment:
missing example

Reply:
could have sworn I've done that already aghhh

Reply:
its not what i meant, you miss third argument, so "Laryntubus" or "Guedeltubus"